src/main/java/seedu/parser/MainParser.java

```diff
@@ -84,12 +91,17 @@ private Command parseEditContact(String userInput) { // userInput is raw user in
         try {
             // split into array of size 3 with command, index and details
             String[] details = userInput.split(" ", NUMBER_OF_EDIT_DETAILS);
```

Comment:
Hi @ng-andre , in the previous `parseAddContact()` function, the `details` array was used for the result of using `contactParser.parseContactDetails()`. I believe what you are trying to do here is not necessarily that, as I can see a `userDetails` array later on for that purpose. I would suggest to keep the naming of variables consistent to avoid confusion.